static/js/components/RootDiskDeviceSelector.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Dispatch, FC, SetStateAction } from "react";
+import { Select } from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { NotificationHelper } from "../types/notification";
+import { queryKeys } from "../util/queryKeys";
+import { LxdDiskDevice } from "../types/device";
+import { fetchStorages } from "../api/storages";
+
+interface Props {
+  notify: NotificationHelper;
+  rootDiskDevice: LxdDiskDevice;
+  setRootDiskDevice: Dispatch<SetStateAction<LxdDiskDevice>>;
+}
+
+const RootDiskDeviceSelector: FC<Props> = ({
```

Comment:
Can we shorten this to `RootDiskSelector`. Actually the component has no notion of root, maybe `StorageSelector`?